packages/suite-web-landing/next.config.js

```diff
@@ -54,6 +54,22 @@ module.exports = withTranspileModules(
                     'process.env.ASSET_PREFIX': JSON.stringify(process.env.ASSET_PREFIX),
                 }),
             );
+            config.module = {
```

Comment:
I think problem is caused by imports using aliases in this package, so imports will be translated into some relative path `../../packages/suite/whatever` instead of using real package style (`import {whatever} from '@trezor/suite'`). That will cause that this file will be considered by compilator as part of this `web-landing` package and not part of `@trezor/suite`. 

And because you don't have defined `@trezor/react-utils` here in `web-landing` package.json, `next-transpile-modules` (defined here line 21) will skip transpilation of this package.

So if my assumption is correct, I think adding `react-utils` to package.json of `web-landing` should solve issue too. Another problem caused by aliases.

Reply:
good, thanks, that helped. 7f04f436608f0b29afdd5e680bfd8a924f879ea5 going to squash fixups I guess. All the stuf that was bothering me should be resolved now